src/elements/content-sidebar/MetadataSidebarRedesign.scss

```diff
@@ -1,5 +1,8 @@
 @import '../common/variables';
+@import '../../styles/variables';
 
 .bcs-MetadataSidebarRedesign {
+    width: 340px;
```

Comment:
This would not be needed if we add `.bcs-content` to `<div className="bcs-MetadataSidebarRedesign">`